src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -51,993 +34,148 @@ def tearDown(self):
-    # Load device info
-    device_c = json.load(
-        open(os.path.join('testcases', 'testdata', 'device_c.json')))
-    # Send grant request
-    grant_0 = json.load(
-      open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-
-    exclusion_zone_db = json.load(open(os.path.join('testcases', 'testdata', 'exclusion_zone_db.json')))
-
-    # Create the actual config.
-    config = {
-        'fccIds': [(device_c['fccId'], 47)],
-        'userIds': [device_c['userId']],
-        'registrationRequests': [device_c],
-        'grantRequests': [grant_0],
-        'expectedResponseCodes': [0],
-        'expectedGrantResponseCodes': [0,400],
-        'expectedHBResponseCodes':[500,501],
-        'expectedRelinquishResponseCodes':[0],
-        'exclusion_zone_db':exclusion_zone_db
-
-    }
-    writeConfig(filename, config)
-
-  def generate_FDB_2_default_config(self, filename):
-    """Generates the WinnForum configuration for FDB.2"""
-    # Load device info
-    device_c = json.load(
+    #Load device info
+    device_a = json.load(
         open(os.path.join('testcases', 'testdata', 'device_a.json')))
-    # Send grant request
+    #Send grant request
     grant_0 = json.load(
       open(os.path.join('testcases', 'testdata', 'grant_0.json')))
-
-    dpa_movelist_db = json.load(open(os.path.join('testcases', 'testdata', 'dpa_db.json')))
-
-    # Create the actual config.
+    #Frequency range 'F2' to modify the exclusion zone database
+    frequency_f2_exz = { 'lowFrequency' : 3680000000,
+                        'highFrequency' : 3690000000 
+                       }
+    #Dictionary containing database information  
+    fake_fcc_db = { 'name': 'Fake_FCC_DB',
+                    'url': 'http://localhost:9090/fakedatabase/exclusionZone',
+                    'databaseFile': 'fakedatabase/exclusion_zone_db.json'
+                  }
+    #Reading the Exclusion zone sample record 
+    data = json.load(
+      open(os.path.join('testcases', 'testdata', 'exz_record_0.json')))
+    #Creating Exclusion zone database
+    writeDB(fake_fcc_db['databaseFile'], data)
+
+    #Getting a random point within the polygon 
+    device_a['installationParam']['latitude'], device_a['installationParam']['longitude'] = getRandomLatLongInPolygon(data)
+    device_a['installationParam']['latitude'] = round(device_a['installationParam']['latitude'],6)
+    device_a['installationParam']['longitude'] = round(device_a['installationParam']['longitude'],6)
+
+    #Frequency range of the grant is adjusted to partially or fully overlap the Exclusion zone frequency range. 
+    grant_0['operationParam']['operationFrequencyRange']['lowFrequency'] =  data['zone']['features'][0]['properties']['freqrange']['lowFrequency']
+    grant_0['operationParam']['operationFrequencyRange']['highFrequency'] = data['zone']['features'][0]['properties']['freqrange']['highFrequency']
+
+    #Create the actual config.
     config = {
```

Comment:
We need to be able to inject Cat B CBSDs as well.

Reply:
Added CatB device   and loaded conditionals